refactor(common): post-review image directive cleanup

pkozlowski-opensource opened the pull request:
Minor cleanups based on the review comments that were
not addressed after the initial review.

Based on https://github.com/angular/angular/pull/47082#pullrequestreview-1074480563

gkalpak commented on packages/common/src/directives/ng_optimized_image/url.ts:
```diff
@@ -24,12 +24,6 @@ export function imgDirectiveDetails(rawSrc: string, includeRawSrc = true) {
   return `The NgOptimizedImage directive ${rawSrcInfo}has detected that`;
 }
```
The `imgDirectiveDetails()` function can also be removed, as it is also defined in [error_helper.ts][1].

[1]: https://github.com/angular/angular/blob/7516bebcd0ffeba45cc14c8e306b988bc71b62be/packages/common/src/directives/ng_optimized_image/error_helper.ts#L10-L14

pkozlowski-opensource replied:
Good catch, thnx!

gkalpak reviewed (commented):
FYI, I have left a couple more minor comments in https://github.com/angular/angular/pull/47082#pullrequestreview-1075647565.

pkozlowski-opensource commented:
Thnx for the additional review @gkalpak - I've converted this PR to a draft to capture your findings.

pkozlowski-opensource committed:
refactor(common): post-review image directive cleanup

Minor cleanups based on the review comments that were
not addressed after the initial review.

kara reviewed (approved):
LGTM

alxhub commented:
This PR was merged into the repository by commit 634e6c7f1ad9492f7ed6cec7aae2dfc132a704a1.
